### PR TITLE
add get CLI command for snapshot locations

### LINKED ISSUE #803: Add `ark snapshot-location get` CLI command

### PR DESCRIPTION
**Pull request by wwitzel3**
Fixes #803

### CODE REVIEW

File: pkg/cmd/cli/snapshotlocation/get.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+				for _, name := range args {
+					location, err := arkClient.Ark().VolumeSnapshotLocations(f.Namespace()).Get(name, metav1.GetOptions{})
+					cmd.CheckError(err)
+					locations.Items = append(locations.Items, *location)
+				}
+			} else {
+				locations, err = arkClient.ArkV1().VolumeSnapshotLocations(f.Namespace()).List(listOptions)
+				cmd.CheckError(err)
+			}
+			_, err = output.PrintWithFormat(c, locations)
```
Comment (skriss) [inline pkg/cmd/cli/snapshotlocation/get.go]: take a look at this method's implementation - you'll need to add a handler for VolumeSnapshotLocations

Comment (wwitzel3) [reply pkg/cmd/cli/snapshotlocation/get.go]: Added, was there something we wanted to display here other than name and provider? I am sure there is but I couldn't really determine what output we wanted from the structs.

File: docs/cli-reference/ark_snapshot-location.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  -n, --namespace string                 The namespace in which Ark should operate (default "heptio-ark")
+      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
+  -v, --v Level                          log level for V logs
+      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging
+```
+
+### SEE ALSO
+
+* [ark](ark.md)	 - Back up and restore Kubernetes cluster resources.
+* [ark snapshot-location get](ark_snapshot-location_get.md)	 - Get snapshot locations
```
Comment (carlisia) [inline docs/cli-reference/ark_snapshot-location.md]: To be consistent, we should make the name of this file `ark_snapshot-locations_get.md` (plural).

File: docs/cli-reference/ark_snapshot-location_get.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+## ark snapshot-location get
+
+Get snapshot locations
+
+### Synopsis
+
+Get snapshot locations
+
+```
+ark snapshot-location get [flags]
```
Comment (carlisia) [inline docs/cli-reference/ark_snapshot-location_get.md]: Should be plural.

File: pkg/cmd/cli/snapshotlocation/get.go [hunk 4]
```diff
@@ -0,0 +1,57 @@
+/*
```
Comment (carlisia) [inline pkg/cmd/cli/snapshotlocation/get.go]: File name should be plural.

File: pkg/cmd/cli/snapshotlocation/snapshot_location.go [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/heptio/ark/pkg/client"
+)
+
+func NewCommand(f client.Factory) *cobra.Command {
+	c := &cobra.Command{
+		Use:   "snapshot-location",
```
Comment (carlisia) [inline pkg/cmd/cli/snapshotlocation/snapshot_location.go]: I think this and the file name should be plural.

File: docs/cli-reference/ark_snapshot-location_get.md (hunk 3 above)
Comment (nrb) [reply docs/cli-reference/ark_snapshot-location_get.md]: These files are autogenerated, based on how the command is registered in cobra.

Comment (carlisia) [reply docs/cli-reference/ark_snapshot-location_get.md]: Thanks!

**Pull request merged**